Source/WebCore/platform/mac/PlatformScreenMac.mm

```diff
@@ -89,19 +89,40 @@ static PlatformDisplayID displayID(Widget* widget)
     ASSERT(hasProcessPrivilege(ProcessPrivilege::CanCommunicateWithWindowServer));
     if (!widget)
         return nil;
-    return widget->platformWidget().window;
+    RetainPtr platformWidget = widget->platformWidget();
+    return [platformWidget window];
```

Comment:
Why not return `[Ref { widget->platformWidget() }.get() window];`?

Reply:
I assume you mean get rid of the local and do:
```
return [RetainPtr { widget->platformWidget() } window];
```
I will make the change.